charts/aerospike/templates/loadgen.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: aerospike-loadgen
+  namespace: default
+spec:
+  schedule: "*/2 * * * *"
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      parallelism: 1
+      backoffLimit: 0
+      template:
+        spec:
+          containers:
+          - name: asbench
+            image: bmedora/aerospike-benchmark:0.0.1
```

Comment:
Can we move this image to this repo?

Reply:
Chicken and egg situation here, sort of. I tested with an image that's built using what's in the PR, but until the PR is merged, CI won't have an image available. 🥚 🐔